src/index.ts

```diff
@@ -59,7 +72,11 @@ export class UnleashClient extends TinyEmitter {
             metricsInterval = 30,
             disableMetrics = false,
             environment = 'default',
-            appName}
+            appName,
+            userId,
+            sessionId,
+            remoteAddress,
+            properties}
```

Comment:
As of now these have no default value. This means when using them later, they could be undefined. I did this in case defaulting them to an empty string would overwrite the defaults in the pipeline somewhere.